packages/stark-core/src/http/http.module.ts

```diff
@@ -1,22 +1,11 @@
 import { NgModule } from "@angular/core";
-import { HttpClient, HttpClientModule } from "@angular/common/http";
+import { HttpClientModule } from "@angular/common/http";
 import { StarkHttpServiceImpl, starkHttpServiceName } from "./services/index";
-import { StarkLoggingService, starkLoggingServiceName } from "../logging/index";
-
-// FIXME: remove this factory once LoggingService and SessionService are implemented
-export function starkHttpServiceFactory(httpClient: HttpClient, starkLoggingService: StarkLoggingService): StarkHttpServiceImpl<any> {
```

Comment:
Now that this factory is removed, the StarkHttpModule requires the Stark Session Service to be defined. That is done by the StarkSessionModule.

This is the reason why the Starter tests are failing in travis: you should add the StarkSessionModule to the `imports` of the testing module in the _home.component.spec.ts_